src/main/sh/mommy

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/env sh
```

Comment:
ironically, this makes the script _less_ portable! the posix standard is unclear on how the shebang line should be interpreted, so giving it more than one argument like you're doing now is open to interpretation. in fact, some operating systems completely ignore everything except the first argument, so they will try to execute the entire script in the "shell" `/bin/env` while ignoring `sh`. oops! in fact, you can see a few of the ci builds failing because of this (e.g. freebsd).

you're theoretically right that not all systems have `sh` at `/bin/sh`, but thus far, i haven't found any.

Reply:
ps: but fixing the shebangs in the completions file is (probably) fine though!